### DISCUSSION TITLE
[Feat Req] Support multiple Auto Intervals

### DISCUSSION THREAD

**Discussion by coding-jj** [grafana/grafana#36987] [Dashboards, Panels & Visualizations]
- What Grafana version are you using?

  dockerhub: grafana/grafana:4.1.2 --- 2017-02-19T12:42:59

- What datasource are you using?

  dockerhub: influxdb:1.2.0-alpine

- What OS are you running grafana on?

  debian:jessie

- What Browser:

  Windows Firefox 52.0 64-bit

- What did you do?

  Added two interval auto intervals:

  - "$interval" with Auto Min Interval 5m
![image](https://cloud.githubusercontent.com/assets/17141243/23900105/2fb30c80-08b8-11e7-95c2-3e707e0bd461.png)

  - "$intervalagent" with Auto Min Interval 1h
![image](https://cloud.githubusercontent.com/assets/17141243/23900450/9f26ae90-08b9-11e7-9975-64d510f77639.png)

- What was the expected result?

  Minimal auto interval for "$interval" is 5m and minimal auto interval "$intervalagent" is 1h. It will work if you set both intervals by hand to a certain value, but not if you leave them on auto.

- What happened instead?

  Minimal Auto interval is not different for both intervals.

  It will be calculated based on the interval that is last in the list of "Templating"
![image](https://cloud.githubusercontent.com/assets/17141243/23900505/e7caeed6-08b9-11e7-8836-edc44e33e8cc.png)

- Background - Use Case 1:
  We have a IBM Domino Statistics with values added to influxdb every 5 Minutes. Some statistic values will only be changed once an hour, so it does not make sense to have a resolution below one hour. 

- Background - Use Case 2:
  In another database the users bad passwords are reported with exact time stamp to influx db. The script that evaluating the logs is run every 15 minutes. So it's not reasonable to have the overall run statistics with an resolution finer than 15 minutes. But it makes sense to zoom in to bad password to minute resolution.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
